Hackney.Shared.PatchesAndAreas/Boundary/Request/GetByPatchNameQuery.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Hackney.Shared.PatchesAndAreas.Boundary.Request
+{
+    public class GetByPatchNameQuery
+    {
+        [FromRoute(Name = "patchName")]
+        public string PatchName { get; set; }
```

Comment:
Could you confirm this indeed is supposed to be `FromRoute` rather than `FromQuery`?

Looking at the ticket and the document for the feature, I could not find the URL example.

Reply:
yeah fromRoute makes more sense because the URL will be `api/v1/patch/patchName/{patchName}`